main.py

```diff
@@ -25,93 +25,32 @@ def __init__(self):
 
         # Used when moving two frames
         self.selected_frame = None
+        self.fullscreen = False
 
     def setup_ui(self):
         """Loads the main.ui file and sets up the window and grid."""
-        self.ui = uic.loadUi("ui/main.ui")
+        self.ui = uic.loadUi("ui/main.ui", self)
         self.grid = VideoFrameGrid()
 
         self.container = self.ui.findChild(QtCore.QObject, "container")
         self.container.addLayout(self.grid)
+        self.menubar = self.ui.findChild(QtCore.QObject, "menuBar")
 
         # Connect up all actions.
         self.actions = {}
         self.actions[MUTE_CHECKBOX] = self.ui.findChild(QtCore.QObject, MUTE_ALL_STREAMS)
-        self.actions[MUTE_CHECKBOX].toggled.connect(self.mute_all_streams)
+        self.actions[MUTE_CHECKBOX].toggled.connect(lambda: mute_all_streams(self))
```

Comment:
Since passing in `self`to the loadUI method above, an error was given when running as `self.mute_all_streams()`. After googling a bit, this seems like the recommended way of doing it. Because of this, I moved the actions to a separate file as we'll, since it then didn't make sense anymore to have it in main.